test/integration/searchRanking.js

```diff
@@ -0,0 +1,207 @@
+const cheerio = require('cheerio');
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const app = require('../../server');
+const constants = require('../../app/lib/constants');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+const resultsRoute = `${constants.SITE_ROOT}/results/`;
+
+function assertSearchResponse(search, assertions) {
+  chai.request(app)
+    .get(resultsRoute)
+    .query({ search })
+    .end(assertions);
+}
+
+function rankingInResults(res, className) {
+  const $ = cheerio.load(res.text);
+  const searchResults = [];
+  for (let i = 0; i < 3; i++) {
+    searchResults.push($(className, $('.results__item--nearby .results__details').eq(i)).text().trim());
+  }
+  return searchResults;
+}
+
+function searchNameRanking(res, expected) {
```

Comment:
I think it would be useful to have this function include the word `expect` to convey this is the assertion part of the test e.g. `expectNameSearchTermInTop3Results`